app/routes/_index.tsx

```diff
@@ -1,19 +1,26 @@
-// import * as Switch from "@radix-ui/react-switch";
+import * as Switch from "@radix-ui/react-switch";
 import type { V2_MetaFunction } from "@remix-run/node";
 import { Link } from "@remix-run/react";
+import { Header } from "../components/header";
 
 export const meta: V2_MetaFunction = () => {
   return [{ title: "New Remix App" }];
 };
 
-// <Switch.Root className="SwitchRoot" id="airplane-mode">
-  //   <Switch.Thumb className="SwitchThumb" />
-// </Switch.Root>
-
 export default function Index() {
   return (
     <main className="h-screen w-screen items-center justify-center flex">
-      <Link to='join' className='hover:bg-gray-100 py-1 px-2 transition-all rounded-md'>Join</Link>
+      <Link
+        to="join"
+        className="hover:bg-gray-100 py-1 px-2 transition-all rounded-md"
+      >
+        Join
+      </Link>
+      <Switch.Root className="SwitchRoot" id="airplane-mode">
+        <Switch.Thumb className="SwitchThumb" />
+      </Switch.Root>
+      <button className="btn btn-primary">Primary button</button>
+      <Header></Header>
```

Comment:
cuando instanciamos un componente sin contenido dentro de el, como es el caso de `Header`, usemos la etiqueta de autocierre. Es mas clara la intencion de esta manera

```suggestion
      <Header />
```